spring-cloud-dataflow-server-kubernetes-docs/src/main/asciidoc/configuration.adoc

```diff
@@ -377,31 +377,184 @@ Request to clean up resources for task execution 1 has been submitted
+
+Using the `boot` Entry Point Style corresponds to using the `exec` style `ENTRYPOINT`. Command line arguments from the deployment request are passed to the container, with the addition of application properties mapped into the `SPRING_APPLICATION_JSON` environment variable rather than command line arguments.
+
+=== Environment Variables
+
+To influence the environment settings for a given application, you can take advantage of the `spring.cloud.scheduler.kubernetes.environmentVariables` property.
+For example, a common requirement in production settings is to influence the JVM memory arguments.
+You can achieve this by using the `JAVA_TOOL_OPTIONS` environment variable, as the following example shows:
+
+====
+[source]
+----
+scheduler.kubernetes.environmentVariables=JAVA_TOOL_OPTIONS=-Xmx1024m
+----
+====
+
+Additionally you can configure environment variables at the server level in the container `env` section of a deployment YAML, as the following example shows:
+
+NOTE: When specifying environment variables in the server configuration and on a per-schedule basis, environment variables will be merged. This allows for the ability to set common environment variables in the server configuration and more specific at the specific schedule level.
+
+====
+[source]
+----
+env:
+- name: SPRING_CLOUD_SCHEDULER_KUBERNETES_ENVIRONMENT_VARIABLES
+  value: myVar=myVal
+----
+====
+
+Replace `myVar=myVal` with your desired environment variables.
+
+=== Image Pull Policy
+
+An image pull policy defines when a Docker image should be pulled to the local registry. Currently, three policies are supported:
+
+* `IfNotPresent`: (default) Do not pull an image if it already exists.
+* `Always`: Always pull the image regardless of whether it already exists.
+* `Never`: Never pull an image. Use only an image that already exists.
+
+The following example shows how you can individually configure containers:
+
+====
+[source,options=nowrap]
+----
+scheduler.kubernetes.imagePullPolicy=Always
+----
+====
+
+Replace `Always` with your desired image pull policy.
+
+You can configure an image pull policy at the server level in the container `env` section of a deployment YAML, as the following example shows:
+
+====
+[source]
+----
+env:
+- name: SPRING_CLOUD_SCHEDULER_KUBERNETES_IMAGE_PULL_POLICY
+  value: Always
+----
+====
+
+Replace `Always` with your desired image pull policy.
+
+=== Private Docker Registry
+
+Docker images that are private and require authentication can be pulled by configuring a Secret. First, you must create a secret in the cluster. Follow the https://kubernetes.io/docs/tasks/configure-pod-container/pull-image-private-registry/[Pull an Image from a Private Registry] guide to create the secret.
```

Comment:
Should second  and third `secret` in paragraph be capitalized?

Reply:
yup, will update on push